lambda_functions/crud/operations.py

```diff
@@ -1,21 +1,79 @@
 import os
-from uuid import uuid4
-from sqlalchemy.orm import relationship, backref, sessionmaker
-from sqlalchemy import create_engine
-from crud.model import Item, User, Review, ReviewAnswer, ReviewQuestion, User, Entity, Keyphrase, Sentiment, URL, ItemEntity, ItemKeyphrase, ItemSentiment, ItemURL, Base, Submission, FactChecking_Organization, ExternalFactCheck
-from datetime import datetime, timedelta
-
+import json
+import random
+import statistics
+import boto3
 import json
 import random
 import statistics
 import sqlite3
 import sys
 import logging
 
+from uuid import uuid4
+from sqlalchemy.orm import relationship, backref, sessionmaker, Session
+from sqlalchemy import create_engine
+from crud.model import Item, User, Review, ReviewAnswer, ReviewQuestion, User, Entity, Keyphrase, Sentiment, URL, ItemEntity, ItemKeyphrase, ItemSentiment, ItemURL, Base, Submission, FactChecking_Organization, ExternalFactCheck
+from datetime import datetime, timedelta
+
+
+
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
 
+
+def set_cors(response, event):
```

Comment:
These functions can now be found in crud/helper.py.

Reply:
Done